tests/unit_tests/test_matchmaker_queue.py

```diff
@@ -34,10 +34,32 @@ def matchmaker_players_all_match():
            Player('Rhiza', player_id=5, ladder_rating=(1500, 50), ladder_games=(config.NEWBIE_MIN_GAMES + 1))
 
 
-def test_newbie_min_games(matchmaker_players):
-    p1, _, _, _, _, p6 = matchmaker_players
-    s1, s6 = Search([p1]), Search([p6])
-    assert s1.ratings[0] == p1.ladder_rating and s6.ratings[0] != p6.ladder_rating
+def test_is_ladder_newbie(matchmaker_players):
+    pro, _, _, _, _, newbie = matchmaker_players
+    assert Search._is_ladder_newbie(pro) is False
+    assert Search._is_ladder_newbie(newbie)
+
+
+def test_is_single_newbie(matchmaker_players):
+    pro, _, _, _, _, newbie = matchmaker_players
+
+    single_newbie = Search([newbie])
+    single_pro = Search([pro])
+    two_newbies = Search([newbie, newbie])
+    two_pros = Search([pro, pro])
+    two_mixed = Search([newbie, pro])
+
+    assert single_newbie.is_single_ladder_newbie()
+    assert single_pro.is_single_ladder_newbie() == False
```

Comment:
Use `is` for singletons like `True`, `False` and `None`
```suggestion
    assert single_pro.is_single_ladder_newbie() is False
```